Jenkinsfile

```diff
@@ -2,50 +2,12 @@
 
 library("govuk")
 
-REPOSITORY = 'govuk_schemas'
-
 node {
-
-  try {
-    stage('Checkout') {
-      checkout scm
-    }
-
-    stage('Clean') {
-      govuk.cleanupGit()
-      govuk.mergeMasterBranch()
-    }
-
-    stage("Set up content schema dependency") {
-      govuk.contentSchemaDependency()
-    }
-
-    stage('Bundle') {
-      echo 'Bundling'
-      sh("bundle install --path ${JENKINS_HOME}/bundles/${JOB_NAME}")
-    }
-
-    stage('Linter') {
-      govuk.lintRuby()
-    }
-
-    stage('Tests') {
-      govuk.setEnvar('RAILS_ENV', 'test')
-      govuk.runTests('spec')
-    }
-
-    if(env.BRANCH_NAME == "master") {
-      stage('Publish Gem') {
-        govuk.publishGem(REPOSITORY, env.BRANCH_NAME)
+  govuk.buildProject(
+    beforeTest: {
+      stage("Set up content schema dependency") {
+        govuk.contentSchemaDependency()
```

Comment:
Is this definitely needed? Looks like it does it regardless in https://github.com/alphagov/govuk-jenkinslib/blame/c64b8c8153613e2e8ae81d8b8df4e20ae080aa9d/vars/govuk.groovy#L157 unless I'm missing something (I probably am)

Reply:
Hmm on second look, I think you're right 👍 will update.